ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/product/v1/product/giftcardProduct.html

```diff
@@ -0,0 +1,126 @@
+            <label for="giftcard_sender_email" class="giftcardProduct__label">
+           <span>${product["giftCardAttributes"]["giftCardOptions"]["Sender Email"]["title"]}</span><span class="required">*</span>
+            <input type="hidden" id="giftcard_sender_email_uid" name="giftcard_sender_email_uid" value='${product["giftCardAttributes"]["giftCardOptions"]["Sender Email"]["value"]["uid"]}'>
+            </label>
+            <div>
+               <input type="email" id="giftcard_sender_email" name="giftcard_sender_email" value="">
+            </div>
+         </div>
+         <div class="giftcardProduct__field">
+            <label for="giftcard_recipient_name" class="giftcardProduct__label">
+              <span>${product["giftCardAttributes"]["giftCardOptions"]["Recipient Name"]["title"]}</span><span class="required">*</span>
+              <input type="hidden" id="giftcard_recipient_name_uid" name="giftcard_recipient_name_uid" value='${product["giftCardAttributes"]["giftCardOptions"]["Recipient Name"]["value"]["uid"]}'>
+            </label>
+            <div>
+               <input type="text" id="giftcard_recipient_name" name="giftcard_recipient_name">
+            </div>
+         </div>
+         <div data-sly-test='${product["giftCardAttributes"]["giftCardType"] == "VIRTUAL" || product["giftCardAttributes"]["giftCardType"] == "COMBINED"}' class="giftcardProduct__field">
+            <label for="giftcard_recipient_email" class="giftcardProduct__label">
+               <span>${product["giftCardAttributes"]["giftCardOptions"]["Recipient Email"]["title"]}</span><span class="required">*</span>
+            <input type="hidden" id="giftcard_recipient_email_uid" name="giftcard_recipient_email_uid" value='${product["giftCardAttributes"]["giftCardOptions"]["Recipient Email"]["value"]["uid"]}'>
+            </label>
+            <div>
+               <input type="email" id="giftcard_recipient_email" name="giftcard_recipient_email" value="">
+            </div>
+         </div>
+         <div class="giftcardProduct__field giftcardProduct__text">
+            <label for="giftcard-message" class="giftcardProduct__label">
+               <span>${product["giftCardAttributes"]["giftCardOptions"]["Message"]["title"]}</span>
+            <input type="hidden" id="giftcard_message_uid" name="giftcard_message_uid" value='${product["giftCardAttributes"]["giftCardOptions"]["Message"]["value"]["uid"]}'>
+            </label>
+            <div>
+               <textarea name="giftcard_message" id="giftcard_message" cols="5" rows="3"></textarea>
+            </div>
+         </div>
+      </fieldset>
+  </div>
+
+	<section class="giftcardProduct__section">
+		<h3>
+			<span class="required">* Required fields</span>
+
+		</h3>
+	</section>
+	<h2 class="productFullDetail__quantityTitle option__title"><span>${'Quantity' @ i18n}</span></h2>
```

Comment:
we should reuse the quantity template here

Reply:
Sure, make sense.